massa-models/src/config/constants.rs

```diff
@@ -122,7 +122,7 @@ pub const MAX_ASYNC_MESSAGE_DATA: u64 = 1_000_000;
 /// Maximum operation validity period count
 pub const OPERATION_VALIDITY_PERIODS: u64 = 10;
 /// cycle duration in periods
-pub const PERIODS_PER_CYCLE: u64 = 128;
+pub const PERIODS_PER_CYCLE: u64 = 10;
```

Comment:
I'm not sure if this change was a decision as part of this feature, or to aid in testing. Prompt to ensure this PR lands with correct periods/cycle count.

Reply:
Yes it's for testing (avoid waiting hours in sandox / labnet just to get a CycleUnavailable :D), I'll add a comment

Reply:
keep it unresolved for now, just so it doesn't slip through. This comment can serve as a checklist-item

Reply:
Resolved now